tests/infra/member.py

```diff
@@ -63,25 +63,17 @@ def set_active(self):
         # Use this with caution (i.e. only when the network is opening)
         self.status = MemberStatus.ACTIVE
 
-    def propose(self, remote_node, script=None, params=None, vote_for=True):
+    def propose(self, remote_node, proposal):
```

Comment:
It seems that we used to be able to pass `vote_for=False` here and now we always pass a vote with the proposal? I'm not sure we even tested this before but perhaps it is something that would be nice to keep?

Reply:
I had a big waffley answer here but it didn't make sense. In short:

It wasn't tested before and I'm not sure the remaining code around it is all correct. I want to take another look in a future PR, with a test for "initial-no-votes", but I'd rather plumb that through separately.

Reply:
Sounds good. This behaviour seems to be tested by some unit tests in `member_voting_test.cpp` so I'm happy to remove this here.